test/functional/app/api/test_credit_transfer_api.py

```diff
@@ -6,23 +6,24 @@
                          "recipient_transfer_accounts_ids_accessor, sender_user_id_accessor,"
                          "recipient_user_id_accessor,"
                          "transfer_type, tier, transfer_status, status_code", [
-    (None, None, lambda t: None, lambda u: None, lambda u: None, lambda u: None, None, 'view', None, 403),
-    (None, None, lambda t: None, lambda u: None, lambda u: None, lambda u: None, None, 'subadmin', None, 403),
+    (None, None, lambda t: None, lambda u: None, lambda u: None, lambda u: None, None, 'view-public', None, 403),
+    (None, None, lambda t: None, lambda u: None, lambda u: None, lambda u: None, None, 'view-private', None, 403),
```

Comment:
I see only one test vector concerning `view-private` in here, but it has been added more places in the api. Should those other additions also have their own vectors?

Furthermore, I can't see a check for the `put` case anywhere.

Reply:
Working on these